examples/guestbook/README.md

```diff
@@ -539,7 +539,7 @@ spec:
     spec:
       containers:
       - name: php-redis
-        image: gcr.io/google_samples/gb-frontend:v3
+        image: gcr.io/google-samples/gb-frontend:v4
```

Comment:
Why is this changed from `google_samples` to `google-samples`? Do we need to change other yaml files too?